scripts/dns-test-execute.bash

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+
+# The integration tests call this to test Linux DNS control, using the `/etc/resolv.conf`
+# method which only works well inside Alpine Docker containers.
+
+set -euo pipefail
+
+HTTPBIN=test.httpbin.docker.local
+
+# Wait for client to ping httpbin (CIDR) resource through the gateway
+docker compose exec -it client timeout 60 \
+sh -c "until ping -W 1 -c 10 172.20.0.100 &>/dev/null; do true; done"
+
+echo "# check original resolv.conf"
+docker compose exec -it client sh -c "cat /etc/resolv.conf.firezone-backup"
+
+echo "# Make sure gateway can reach httpbin by DNS"
+docker compose exec -it gateway sh -c "curl $HTTPBIN/get"
+
+echo "# Try to ping httpbin as a DNS resource"
```

Comment:
```suggestion
echo "Try to ping httpbin as a DNS resource"
```

These are all not necessary, right? :)

Reply:
I thought they'd make the debug logs in Github a little easier to read